documentation/asciidoc/topics/yaml/cr_backup_site_automatic.yaml

```diff
@@ -19,4 +19,4 @@ spec:
   logging:
     categories:
       org.jgroups.protocols.TCP: error <10>
-      org.jgroups.protocols.relay.RELAY2: fatal <11>
```

Comment:
@ryanemerson sorry, just for information, is this problem was detected by CRD OpenAPI schema validation on CR create or you find this manually?

Reply:
I guess it was the schema validation. I got the following error when I tried to create the CR:

```
The Infinispan "datagrid" is invalid: spec.logging.categories.org.jgroups.protocols.relay.RELAY2: Unsupported value: "fatal": supported values: "trace", "debug", "info", "warn", "error"
```